odbc/src/api/col_attr_describe_tests.rs

```diff
@@ -15,10 +15,10 @@ mod unit {
     // has been called).
```

Comment:
All test files should be safely ignorable, the only change is to creating handles (and some memory cleanup).